src/main/java/co/inventorsoft/academy/collections/model/Range.java

```diff
@@ -1,60 +1,119 @@
 package co.inventorsoft.academy.collections.model;
 
+import java.util.ArrayList;
 import java.util.Collection;
 import java.util.Iterator;
+import java.util.List;
 import java.util.Set;
+import java.util.function.Function;
+
 
 public class Range<T> implements Set<T> {
 
+    List<T> mylist  = new ArrayList();
+
+
+    public Range(float a, float b) {
+        if(!(a==b)){
+            while (a <=b) {
+                mylist.add((T)(Float) a);
+                a = a + 0.1f;
+            }}
+    }
+
+    public Range(int a, int b) {
+        if (!(a==b)){
+            while (a <= b) {
+                mylist.add((T)(Integer)a);
+                a = a + 1;
+            }}
+
+    }
+
+
+    public static Range of(int i, int i1) {
+        return new Range(i, i1);
+    }
+
+    public static Range of(float i, float i1) {
+        return new Range(i, i1);
+    }
     public int size() {
-        return 0;
+        return mylist.size();
     }
 
     public boolean isEmpty() {
-        return false;
+        return mylist.isEmpty();
     }
 
+    @Override
     public boolean contains(Object o) {
-        return false;
+        return mylist.contains(o);
     }
 
-    public Iterator<T> iterator() {
-        return null;
+    @Override
+    public Iterator iterator() {
+        return mylist.iterator();
     }
 
+    @Override
     public Object[] toArray() {
-        return new Object[0];
+        return mylist.toArray();
     }
 
-    public <T1> T1[] toArray(T1[] a) {
-        return null;
+    @Override
+    public boolean remove(Object o) {
+        return mylist.remove(o);
     }
 
-    public boolean add(T t) {
-        return false;
+    public boolean addAll(Collection<? extends T> c) {
+        for (T t : mylist) {add(t);}return false;
```

Comment:
Format code. Why does it always return false?

Reply:
corrected